xero_python/api_client/oauth2.py

```diff
@@ -278,6 +278,7 @@ def update_token(
         expires_at=None,
         refresh_token=None,
         id_token=None,
+        **kwargs
```

Comment:
@j-osephlong please remove the unused parameter

Reply:
Just fixed, please see new commit [593d372](https://github.com/XeroAPI/xero-python/pull/132/commits/593d37297b124c1ea8810ec5cf6e4d1efddbe602)